csv.md

```diff
@@ -0,0 +1,124 @@
+
+Example measures include counts (e.g. of people, currency, companies), indexes, and frequency.
+
+#### Unit columns
+
+Unit columns are used to describe the unit of measurement for the observation when a dataset employes multiple units. The majority of the time there will be a single unit for each measure, but that is not always the case. Units are critical to comparability, allowing for conversions.
+
+Example units include kilograms, miles, numbers, and percentages.
+
+**Note:** Count is not a unit. It is a measure. The unit you want is "number".
+
+**Note:** Indicies are only comparable to itself, therefore they are "unitless". You can compare miles and kilometres, but you cannot compare the FTSE 100 and the FTSE 250 values directly you can only compare their trends.
+
+The units used should *always* be QUDT units unless no appropriate unit exists, which is exceptionally rare. Again, more often in statistics you're looking at numbers, rates, percentages, fractions, and frequencies.
+
+##### Scaling units
+
+Often in summary statistics publications units need to be scaled in order to control disclosure, or to maintain an appropriate level of granularity. Scaled units are especially common in publications like National Accounts, which contains the caucluated and reported spend of the Government of the United Kingdom and Northern Ireland, which totals over a trillion GBP (i.e. 1000000000000) and those figures are hard to sight read.
+
+When scaling units take the base unit and suffix the multiplication factor preceeded by an underscore. Written and decimal representations are both acceptable For several examples,
+
+- `GBP_millions` for millions of GBP (used in National acounts)
+- `NUM_1000` for thousands (used in SOME PUBLICATION)
+- `PER_tenths` (i.e. `PER`cent * 0.1 a.k.a tenths) for per thousands (used in SOME PUBLICATION)
+- `ratio_0.001` for per thousands (used in SOME PUBLICATION)
+- `L_100` for hectolitres (used in HMRC Alcohol Bulletin)
+
+### Ordering
+
+Ensuring that users can understand your CSV files is important. This is especially true when the
+
+###
+
+Concept Clarity: Ensure that the concepts used in the CSV files are clear and easily understandable to enhance human readability.
+
+Unique Addressability: Each observation should be uniquely addressable by filtering all columns (except the observation column) to a value.
+
+Column Order: Columns should be ordered as follows: time period, geographic region, and then by descending count of options in each column.
+
+Value Completeness: All columns should have values for every observation, except for the observation, observation status, or attribute type columns.
+
+Next steps could include:
+
+Implementing these guidelines in your CSV file creation process.
```

Comment:
I'd use bullet points or a numbered list for these steps just to make it more readable.